python/lsst/validate/drp/base.py

```diff
@@ -17,9 +17,22 @@
 # You should have received a copy of the LSST License Statement and
 # the GNU General Public License along with this program.  If not,
 # see <https://www.lsstcorp.org/LegalNotices/>.
+"""lsst.validate.drp's Measurement API that handles JSON persistence.
+"""
 
```

Comment:
What would you want moved from `base` to `io`? Just the JsonSerializationMixin?

If that's it, it might make sense to fork the measurement framework into a new package (lets call it `squashkit` as an example) and then put the following code objects at the root namespace:

```
from squashkit import (Metric, Specification, MeasurementBase, BlobBase, Datum, Job)
```

(And probably provide exception classes as well for things like `ValidateErrorSpecification` and `ValidateErrorUnknownSpecificationLevel`)

No one really needs to know about the internal mixins like DatumAttributeMixin and JsonSerializationMixin.